src/modular/boxed_monty_form/invert.rs

```diff
@@ -22,22 +22,8 @@ impl BoxedMontyForm {
     }
 }
 
-impl Invert for BoxedMontyForm {
```

Comment:
I think this should still implement Invert?

Reply:
Whoops, good catch